app/notifications/reimbursement_complete_notification.rb

```diff
@@ -16,6 +16,6 @@ def message
   end
 
   def url
-    case_contacts_path(casa_case_id: params[:case_contact].casa_case_id)
```

Comment:
From what I can tell there seems to be a mix of styles in the existing code of `params[:some_object][:foo]` and `params[:some_object].foo` I would rather make them all `params[:some_object].foo`. Otherwise it feel like we are dealing with hashes not active record objects. 

This applies to all the places with `params[:something][:something_else]`